src/rest_vol_debug.c

```diff
@@ -435,6 +435,30 @@ file_specific_type_to_string(H5VL_file_specific_t specific_type)
     } /* end switch */
 } /* end file_specific_type_to_string() */
 
+/*-------------------------------------------------------------------------
+ * Function:    file_optional_type_to_string
+ *
+ * Purpose:     Helper function to convert each member of the
+ *              H5VL_file_optional_t enum into its string representation
+ *
+ * Return:      String representation of given object or '(unknown)' if
+ *              the function can't determine the type of object it has
+ *              been given (can't fail).
+ *
+ * Programmer:  Matthew Larson
+ *              January, 2024
+ */
+const char *
+file_optional_type_to_string(H5VL_optional_args_t optional_type)
```

Comment:
Why not just pass an H5VL_file_optional_t here so you don't need to define an enum value to determine which operation is being passed in?